src/app/components/extendedScreenContainer/index.tsx

```diff
@@ -1,17 +1,18 @@
 import useWalletSelector from '@hooks/useWalletSelector';
+import { useMemo } from 'react';
 import { useTranslation } from 'react-i18next';
-import { useDispatch } from 'react-redux';
 import { Outlet } from 'react-router-dom';
 import styled from 'styled-components';
 
-const ExtendedScreenRouteContainer = styled.div((props) => ({
+interface Props {
+  isGalleryOpen: boolean;
+}
+const ExtendedScreenRouteContainer = styled.div<Props>((props) => ({
   display: 'flex',
   flexDirection: 'column',
-  height: '100vh',
-  width: '100vw',
+  height: props.isGalleryOpen ? '100vh' : '100%',
+  width: props.isGalleryOpen ? '100vw' : '100%',
```

Comment:
will want to test this doesn't break all screens which use ExtendedScreenRouteContainer